portfolio/src/main/webapp/index.html

```diff
@@ -10,9 +10,26 @@
     <div id="content">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+	<img src="images/profile.jpg"></img>
+	<p>Hello, I'm Savion. I'm an incoming junior at Columbia University.</p>
+	<p>Projects I have worked on.</p>
+	<ul>
+		<li><b>Kabloombox</b>: a web app that recommends foreign-language songs to you based off of your music preference.</li>
+		<li><b>Mix.me</b>: an iOS app that creates a customizeid playlist of a certain length meant to be used as an alternative to a traditional timer or alarm clock</li>
+		<li><b>Juxtaposts</b>: a web app that allows you to easily compare articles from 2 different news sources on a similar topic</li>
+	</ul>
+	<a href="https://www.linkedin.com/in/savs">Find me on LinkedIn</a><br />
```

Comment:
`<br>` instead of `<br />`: https://google.github.io/styleguide/htmlcssguide.html#Document_Type